data/validate_metadata_docu_files.js

```diff
@@ -0,0 +1,42 @@
+const fs = require('fs')
+
+const meta = require('./metadata.json')
```

Comment:
Why not parse the JSON directly? Not tested, but:
```suggestion
const meta = JSON.parse(fs.readFileSync('./metadata.json', 'utf8'))

Reply:
I took over your suggestion for reading all JSON files in those tests. I don't think there are any major differences except for the caching `require` does and I doubt that this would become a problem here. However, lets better be save than sorry.

Reply:
I thought that requiring a json file parses it, no?

Reply:
We can always easily see for ourselves (that is so nice about python, ES, et al.):
```javascript
some/where via  v16.14.2
❯ node
Welcome to Node.js v16.14.2.
Type ".help" for more information.
> const no_way = require('./invalid')
Uncaught:
SyntaxError: /some/where/invalid.json: Unexpected token n in JSON at position 9
```
given the JSON file `invalid.json`:
```json
{"boom: "no way"}
```

Reply:
@sthagen That seems to be the same Uncaught SyntaxError in both variants...

Reply:
@valentindey: @domachine uses `const json = JSON.parse(await readFile(filePath, { encoding: 'utf-8' }))` in the csaf-validator-lib...

Reply:
> @sthagen That seems to be the same Uncaught SyntaxError in both variants...

Two different concerns: 1) Availability of external resource 2) implicit parsing or not.

To show that given a valid and available JSON file node implicitly parses per `require`:
```javascript
some/where via  v16.14.2
❯ echo '{"foo": "bar"}' > valid.json

some/where via  v16.14.2
❯ node
Welcome to Node.js v16.14.2.
Type ".help" for more information.
> const parsed = require('./valid')
undefined
> parsed.foo
'bar'
> typeof(parsed)
'object'
```

Reply:
As both can do the same, the question remains: Is one better than the other? If so, why/how? If not than it doesn't make a difference...

> I thought that requiring a json file parses it

Thanks for pointing that out, I didn't know that.

Reply:
> @valentindey: @domachine uses `const json = JSON.parse(await readFile(filePath, { encoding: 'utf-8' }))` in the csaf-validator-lib...

I used it there because [json imports](https://nodejs.org/api/esm.html#json-modules) in es modules are still experimental. Since this is a commonjs-script, require seems alright for me.

Reply:
I wouldn't call myself a JS expert, but thanks for your trust :slightly_smiling_face: I don't see any reason to now change it back.